### ISSUE TITLE
examples in the book are broken

### ISSUE THREAD

**Issue by reuvenpo** [sagan-software/eosio-rust#15]
all the links under this page are broken:
https://sagan-software.github.io/eosio-rust/examples/index.html
and all the subpages are empty

**Comment by Hedzer** [comment]
I came here to say the same thing.

**Comment by liamcurry** [comment]
Hey sorry the book is a work-in-progress -- I'll be updating it with better instructions and working examples later this week.

**Comment by kegesch** [comment]
I wanted to try out this crate today. Unfortunately also the "Getting Started" example does not work with version `0.3` anymore.

**Comment by liamcurry** [comment]
Yes sorry the book is very outdated. For now the best way to get started would be to look at the contracts in the `contracts` and `examples` folders.
